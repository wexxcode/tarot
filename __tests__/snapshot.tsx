import { act, render } from '@testing-library/react'
import Home from '@/pages/index'

describe('Home component', () => {
  it('renders homepage unchanged', () => {
    let container
    act(() => {
      ({ container } = render(<Home />))
    })
    expect(container).toMatchSnapshot()
  })
})