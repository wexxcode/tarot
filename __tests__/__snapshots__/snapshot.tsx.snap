// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home component renders homepage unchanged 1`] = `
<div>
  <div
    class="container container-fluid"
  >
    <nav>
      <h2>
        MEU TAROT
      </h2>
    </nav>
    <main>
      <p>
        Carregando... Esperando a resposta do universo
      </p>
    </main>
  </div>
</div>
`;
